apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx

```diff
@@ -54,147 +120,680 @@ const SignUpPage = () => {
+                id: "6SfJEp",
+                description:
+                  "Instructions on how to sign in with GCKey - IAP variant",
+              })}
             </p>
-          </div>
+            <p data-h2-margin="base(x.5, 0, 0, 0)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "If you're unsure of whether you have an existing GCKey account, continue to the website and try signing in. If you can't remember your password, you can also reset it there.",
+                id: "Z+gi2d",
+                description:
+                  "Instructions on what to do if user doesn't know if they have a GCKey - IAP variant",
+              })}
+            </p>
+            <p data-h2-margin="base(x.5, 0, 0, 0)">
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "<strong>Don't have a GCKey account?</strong> <a>Sign up for one</a>.",
+                  id: "QsWizb",
+                  description:
+                    "Instruction on what to do if user does not have a GCKey",
+                },
+                {
+                  a: (chunks: React.ReactNode) =>
+                    buildExternalLink(loginPath, chunks),
+                },
+              )}
+            </p>
+            <p data-h2-margin="base(x1, 0, 0, 0)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "<strong>Heads up!</strong> Before we send you off to set up your GCKey credentials, we ask that you take another look at the program's eligibility criteria to ensure that you meet the requirements.",
+                id: "ow71P8",
+                description:
+                  "A request for the user to review the IAP eligibility criteria",
+              })}
+            </p>
+            <p data-h2-margin="base(x2, 0, 0, 0)">
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "If you have questions concerning this step, or if you are unsure about how to proceed, please feel free to reach out to our support team at <a>edsc.pda-iap.esdc@hrsdc-rhdcc.gc.ca</a>.",
+                  id: "ltzA7w",
+                  description:
+                    "How to get help from the support team - IAP variant",
+                },
+                {
+                  a: (chunks: React.ReactNode) =>
+                    buildExternalLink(
+                      "mailto:edsc.pda-iap.esdc@hrsdc-rhdcc.gc.ca",
+                      chunks,
+                    ),
+                },
+              )}
+            </p>
+          </>
+        )}
+
+        <hr data-h2-margin="base(x2, 0)" />
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column) l-tablet(row)"
+          data-h2-align-items="base(center)"
+          data-h2-justify-content="base(space-between)"
+        >
```

Comment:
We should add either a gap value here or margin to the paragraph tags. Looks a little tight on mobile.

![Screenshot 2023-09-08 150909](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/742e0e6d-b788-418f-8d32-88a3c38906ab)

Reply:
Good catch!  Contributed to the other branch.
[styling tweaks from dev review](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7835/commits/8fabb396dca6bcbf0dc5e32332e2e73c3034efda)